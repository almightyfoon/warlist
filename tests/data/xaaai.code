c9101b_-0LgO6reFeFeF6U4x4x5GfI5V2NhV___-0I3Z6r4Cn66N6un2n26vn3n3n9n1n9n1mBmB
