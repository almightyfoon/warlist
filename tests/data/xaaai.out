https://conflictchamber.com/#c9101b_-0LgO6reFeFeF6U4x4x5GfI5V2NhV___-0I3Z6r4Cn66N6un2n26vn3n3n9n1n9n1mBmB

Skorne Army - 75 / 75 points
[Theme] Imperial Warhost

[Zaadesh 2] Lord Tyrant Zaadesh [+28]
 - Agonizer [0(6)]
 - Aradus Soldier [16]
 - Aradus Soldier [16]
 - Aradus Soldier [16]
 - Archidon [10]
 - Basilisk Krea [0(7)]
 - Basilisk Krea [0(7)]
 - Titan Gladiator [15]
Bog Trog Mist Speaker [4]
Paingiver Beast Handlers (max) [7]
Swamp Gobber Bellows Crew [2]
Siege Animantarax [17]

Skorne Army - 75 / 75 points
[Theme] The Exalted

!!! This army contains pre-release entries.

[Morghoul 2] Lord Assassin Morghoul [+29]
 - Agonizer [6]
 - Cyclops Shaman [8]
 - Titan Cannoneer [16]
 - Aptimus Marketh [0(5)]
Ancestral Guardian [0(5)]
Extoller Novitiate [2]
Extoller Novitiate [2]
Extoller Soulward [0(3)]
Immortal Vessel [3]
Immortal Vessel [3]
Immortals (max) [13]
 - Extoller Advocate [3]
Immortals (max) [13]
 - Extoller Advocate [3]
Supreme Guardian [16]
Supreme Guardian [16]
